scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -56,13 +56,14 @@ def get_machine_arch():
     return os.uname()[4]
 
 
-def get_os_loader():
-    path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+def get_os_loader(firmware_type=None):
+    path = None
+    template = "<loader readonly='yes' type='pflash'>%s</loader>"
     if 'aarch64' in get_machine_arch():
-        return """
-  <loader readonly='yes' type='pflash'>%s</loader>
```

Comment:
at least for the aarch64 case we actually still need the <nvram> element (and you also need it for the uefi case, otherwise reboot does not work)